samples/grids/data-grid/column-types/package.json

```diff
@@ -12,10 +12,11 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "@infragistics/igniteui-react-core": "20.1.0-dev.586",
-    "@infragistics/igniteui-react-charts": "20.1.0-dev.586",
-    "@infragistics/igniteui-react-grids": "20.1.0-dev.586",
-    "@infragistics/igniteui-react-inputs": "20.1.0-dev.586",
+    "@infragistics/igniteui-react-core": "20.1.0-dev.584",
+    "@infragistics/igniteui-react-charts": "20.1.0-dev.584",
+    "@infragistics/igniteui-react-grids": "20.1.0-dev.584",
+    "@infragistics/igniteui-react-layouts": "20.1.0-dev.584",
+    "@infragistics/igniteui-react-inputs": "20.1.0-dev.584",
```

Comment:
undo changes to this file because v20.1.0-dev.586 is newer

Reply:
Updated to 586
https://github.com/IgniteUI/igniteui-react-examples/pull/22/commits/57011f7d5e26fa323bc63c8e1de2bdaa1b8f16b5